packages/web/src/common/store/lineup/sagas.ts

```diff
@@ -378,7 +382,8 @@ function* fetchLineupMetadatasAsync<T extends Track | Collection>(
       const deletedCount =
         lineupMetadatasResponse.length -
         responseFilteredDeletes.length -
-        nullCount
+        nullCount +
+        duplicateCount
```

Comment:
sure it's `+ duplicateCount` and not `- duplicateCount`? not sure how this value is used later.. so many counts

Reply:
yes i'm sure. this math is weird but i don't want to break it more